src/components/SkipToMainContent/styles.css

```diff
@@ -0,0 +1,23 @@
+.skipToMainContent {
+  background-color: var(--color-hunter-green);
+  color: var(--color-white);
+  font-weight: 600;
```

Comment:
What do you think about adding the font weights to css variables?

Reply:
yes, that is something I have pending. I will do it in a separate PR to change it everywhere.